apps/main/src/dao/components/PageAnalytics/VeCrvFeesTable/index.tsx

```diff
@@ -43,13 +41,13 @@ const VeCrcFees: React.FC = () => {
               <>
                 <FeesContainer>
                   {veCrvFees.fees.map((item) => (
-                    <FeeRow key={item.date}>
+                    <FeeRow key={item.timestamp.getTime()}>
                       <FeeDate>
-                        {item.date}
-                        {item.timestamp > currentTime && <span> {t`(in progress)`}</span>}
+                        {formatDateFromTimestamp(convertToLocaleTimestamp(item.timestamp.getUTCTimestamp()))}
+                        {item.timestamp.getTime() > Date.now() && <span> {t`(in progress)`}</span>}
```

Comment:
Again, no timestamps needed. Dates may be compared directly
```suggestion
                        {formatDate(item.timestamp)}
                        {item.timestamp > new Date() && <span> {t`(in progress)`}</span>}
```
```